SwiftReflector/MarshalEngineCSafeSwiftToCSharp.cs

```diff
@@ -222,9 +222,15 @@ public MarshalEngineCSafeSwiftToCSharp (CSUsingPackages use, List<string> identi
 						argTypes [idx] = parmType.GenericTypes [idx].Typeof ();
 					}
 					var typeArr = new CSArray1DInitialized (CSSimpleType.Type, argTypes);
+					CSBaseExpression flags = new CSIdentifier ("ClosureFlags.None");
+					var closure = swiftParm.TypeSpec as ClosureTypeSpec;
+					if (closure.Throws)
```

Comment:
So we could get None | Throws | Async as a valid value here?

Reply:
Yes. It compiles. I checked.